<h4>Lunar Lander</h4>

A lunar lander is free-falling toward the moon, and at an altitude of $53$ kilometers above the
lunar surface its downward velocity is measured at $1477\,km/h$. Its retrorockets, when fired
in free space, provide a deceleration of $T=4\,m/s^{2}$. At what height above the lunar surface
should the retrorockets be activated to ensure a soft touchdown ($v=0$ at impact)?

$M$ is the mass of the moon. $R$ is the radius of the moon, $G$ is the gravitational constant.

In [1]:
M=7.35e22

7.35e22

In [2]:
R=1.74e6

1.74e6

In [3]:
G=6.6726e-11

6.6726e-11

In [10]:
v2=1477*1000/3600

410.27777777777777

In [11]:
t1=v2/4

102.56944444444444

In [7]:
(-2)*t1^2+410*t1

21012.49035493827

Without gravity, $s_{1}$ is about $21\,km$, the height at which the retrorockets should be fired. 

$$
\frac{d^{2}r}{dt^{2}}=T-\frac{GM}{r^{2}}
$$
implies
$$
\frac{1}{2}v^{2}=\frac{GM}{r}+Tr+C
$$
since $v(0)=0$,
$$
C_{1}=-\frac{GM}{R}-4R
$$

In [9]:
C1=((-G*M)/R)-4*R

-9.778598275862068e6

and 
$$
\frac{1}{2}v^{2}=\frac{GM}{r}+4r+C_{1}
$$
after ignition. Before ignition, we know that
$$
\frac{1}{2}v_{2}^{2}=\frac{GM}{R+53000}+C_{2}$$

In [12]:
C2=0.5*v2^2-((G*M)/(R+53000))

-2.651118281119821e6

At $t_{1}$, the two functions agree, so
$$
\frac{GM}{r(t_{1})}+C_{2}=\frac{GM}{r(t_{1})}+4r(t_{1})+C_{1}
$$
and therefore

In [14]:
r1=(C2-C1)/4

1.7818699986855616e6

In [15]:
r1-R

41869.998685561586

The retrorockets should be fired at $41870$ metres above the lunar surface. Comment: I spent some futile time calculating $r(t)$, which isn't necessary in this case since the initial conditions are conveniently provided in terms of $v(t)$.
